assembly/src/tokens/mod.rs

```diff
@@ -242,7 +267,12 @@ impl<'a> Token<'a> {
         assert_eq!(Self::REPEAT, self.parts[0], "not a repeat");
         match self.num_parts() {
             0 => unreachable!(),
-            1 => Err(ParsingError::missing_param(self)),
+            1 => Err(
+                    ParsingError::missing_param(
+                        self,
+                        "repeat <num_repetitions>",
+                    )
```

Comment:
The correct format here would be `repeat.<num_repetitions>`.

Reply:
Changed.